garden_ai/app/garden.py

```diff
@@ -59,19 +33,6 @@ def garden():
 
 @garden_app.command(no_args_is_help=True)
 def create(
-    directory: Path = typer.Argument(
-        None,
-        callback=setup_directory,
-        dir_okay=True,
-        file_okay=False,
-        writable=True,
-        readable=True,
-        resolve_path=True,
-        help=(
-            "(Optional) if specified, this generates a directory with subfolders to help organize the new Garden. "
-            "This is likely to be useful if you want to track your Garden/Pipeline development with GitHub."
-        ),
-    ),
```

Comment:
@OwenPriceSkelly You mentioned in the summer student meeting that you were open to having students get rid of this ... so I took that as a signal that it was ok to get rid of this while I was in here.

Reply:
oh yeah more than ok getting rid of this part now